ecs-agent/daemonimages/csidriver/Makefile

```diff
@@ -1,27 +1,40 @@
-CSI_DRIVER_VERSION="v$(shell cat ./container/VERSION)"
-GO111MODULE=on
-GOPATH=$(shell go env GOPATH)
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+.PHONY: all image mockgen clean
+
+CSI_DRIVER_VERSION="v$(shell cat CSI_DRIVER_VERSION)"
 BUILD_DATE=$(shell date -u -Iseconds)
+GO_VERSION=$(shell cat ../../../GO_VERSION)
 
-OS?=linux
-ARCH?=amd64
```

Comment:
We also need to support ARM

Reply:
this is automatically determined by the build environment, we don't cross-compile our production builds

Reply:
TIL. Ok, as you can see the arch is used here `CGO_ENABLED=0 GOOS=$(OS) GOARCH=$(ARCH) go build -ldflags`. As long as this can be detected automatically, then it should be fine. Thanks.

Reply:
Explicitly setting GOARCH I think can be risky in a docker build situation, since the `docker pull` for the eks csi base image will pull down whatever arch the build env is. So if you set GOARCH=x86_64 inside a aarch64 build env, you'd end up with an x86_64 cross-compiled binary trying to run inside an aarch64 container, which would obviously fail to ever start.